vent/controller/control_module.py

```diff
@@ -555,6 +574,10 @@ def heartbeat(self):
         '''only used for fiddling'''
         return self._loop_counter
 
+    @property
+    def running(self):
```

Comment:
Are is_running and .running are duplicated?

Reply:
yes, i had added this awhile ago before the is_running was there, but we don't need it and it isn't used anywhere. i'll let you pick which you'd rather have :p